src/content/getting-started/quickstart/kind/_index.md

```diff
@@ -29,6 +29,8 @@ make deploy
 By default, the automation configuration in the submariner-io/submariner-operator repository deploys two clusters, with cluster1 configured as
 the Broker. See the [settings](https://github.com/submariner-io/submariner-operator/blob/devel/.shipyard.e2e.yml) file for details.
 
+Please note: the issue of running out of inotify resources can be fixed by following the [known-issues guide](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files)
```

Comment:
There's a built-in for formatting notices, maybe it would be good to do something like:

```suggestion
{{% notice note %}}
You may need to [increase your inotify resource limits](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files).
{{% /notice %}}
```

The automation should also warn about this and provide docs about how to fix it. Did that work for you?

https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/clusters.sh#L206

Reply:
This might also fit as a general kind prerequisite instead of a notice in automatic deploy section, as it applies to both manual and automatic deployments.

Reply:
The automation did warn and print a note about the issue. But to be fair, I didn't spot it right away :$. Just thought it would be a handy thing to note upfront.

I will update the note to use the notice format and move it into the prerequisite section.

Reply:
> The automation did warn and print a note about the issue. But to be fair, I didn't spot it right away :$. Just thought it would be a handy thing to note upfront.

Yep, agreed and thanks!